apps/wasm-graphcompiler-tvm/wasm-graphcompiler/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "wasm-graphcompiler-tvm"
+version = "0.1.0"
+authors = ["leonwanghui <wanghui71leon@gmail.com>"]
```

Comment:
let us rename authors to TVM contributors, the rationale is that the same code will be contributed by multiple contributors, and the contribution is already recored by the commit history.